src/app/screens/Library/layout.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { array } from 'prop-types';
+import { FlatList } from 'react-native';
+import { useHeaderHeight } from '@react-navigation/stack';
+
+import withLoader from '@components/Loader';
+import { keyExtractor } from '@utils/renders';
+import { POLAR } from '@constants/colors';
+
+import { renderListItem } from './utils';
+import styles from './styles';
+
+function Library({ books }) {
+  const headerHeight = useHeaderHeight();
+  const padding = styles.bookList.paddingTop;
+
+  return (
+    <FlatList
+      style={styles.bookList}
+      data={books}
+      renderItem={renderListItem}
+      keyExtractor={keyExtractor}
+      contentContainerStyle={{ ...styles.bookListContainer, paddingTop: headerHeight + padding }}
+    />
+  );
+}
+
+Library.propTypes = {
+  books: array
```

Comment:
when we define the prop of an array element, we usually use `arrayOf` in prop-types to define what are the element's types
But don't worry because we are using typescript now

Reply:
Awesome! I forgot to change it, will do so on typescript 🚀